scripts/case_selection/export_bpc_selected_cases.R

```diff
@@ -165,6 +165,7 @@ sample_info_list <- lapply(samples_per_patient,function(x){
     temp_df$cpt_genie_sample_id = x[i]
     temp_df$cpt_oncotree_code = clinical$oncotree_code[clinical$sample_id == x[i]]
     temp_df$cpt_sample_type = clinical$sample_type_detailed[clinical$sample_id == x[i]]
+    temp_df$cpt_sample_type = sample_type_mapping$CODE[match(temp_df$cpt_sample_type, sample_type_mapping$DESCRIPTION)]
```

Comment:
Can you speak to why this was changed here?

Reply:
Since as the Jira ticket, we would like to output numeric value for this column:



Input: Main GENIE [Value] in Consortium Release | Output: BPC [Numerical Code] in Export File
-- | --
PRIMARY_RACE_DETAILED | naaccr_race_code_primary
SECONDARY_RACE_DETAILED | naaccr_race_code_secondary
TERTIARY_RACE_DETAILED | naaccr_race_code_tertiary
SEX_DETAILED | naaccr_sex_code
ETHNICITY_DETAILED | naaccr_ethnicity_code
SAMPLE_TYPE_DETAILED | cpt_sample_type

Reply:
Thanks, but I thought the previous line covered that: 
```
clinical$sample_type_detailed[clinical$sample_id == x[i]]
```

Reply:
Ohhh interesting, before we weren't providing these as codes.  @Chelsea-Na , I wonder... if SAMPLE_TYPE_DETAILED is meant to be provided as a string value instead

Reply:
The `sample_type_detailed` renders info in DESCRIPTION column in this [table](https://www.synapse.org/Synapse:syn7434273/tables/).

Reply:
Hmmm yeah, I'm not sure why we were providing `SAMPLE_TYPE` as the Description value instead of the numerical code. I would recommend we just stick to how we have always done provided it (using the Description value).